app/src/main/java/com/sopt/instagram/data/model/response/wrapper/BaseResponse.kt

```diff
@@ -12,5 +12,5 @@ data class BaseResponse<T>(
     @SerialName("message")
     val message: String,
     @SerialName("data")
-    val data: T? = null,
+    val data: T
```

Comment:
```suggestion
    val data: T? = null,
```
data가 전달되지 않는 응답도 존재하기 때문에 nullable하게 설정 부탁드려요!

Reply:
넵!!